### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman100**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

## Further comments

Not sure, how to resize the logo to 256x256 pixel

### CODE REVIEW

File: minecraft_servers/archmc/logo.png
Comment (DoJapHD) [inline minecraft_servers/archmc/logo.png:1]: File is actually a jpg, convert to png

File: minecraft_servers/archmc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "server_name": "archmc",
+    "nice_name": "ArchMC",
+    "direct_ip": "mc.archmc.lol",
+    "server_wildcards": [
+      "%.laby.net"
```
Comment (DoJapHD) [inline minecraft_servers/archmc/manifest.json:6]: Either remove or add your own wildcard

File: minecraft_servers/archmc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "supported_languages": [
+       "en"
+    ],
+    "social": {
+      "web": "https://arch.lol",
+      "web_shop": "https://arch.lol/store",
+      "tiktok": "ruleyourowngame",
+      "discord": "https://discord.gg/labymod",
```
Comment (DoJapHD) [inline minecraft_servers/archmc/manifest.json:15]: Either remove or add your own

File: minecraft_servers/archmc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "server_name": "archmc",
+    "nice_name": "ArchMC",
+    "direct_ip": "mc.archmc.lol",
```
Comment (DoJapHD) [inline minecraft_servers/archmc/manifest.json:4]: Server is offline

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/archmc/manifest.json (hunk 3 above)
Comment (Justman100) [reply minecraft_servers/archmc/manifest.json]: See 0b335ad

File: minecraft_servers/archmc/manifest.json (hunk 2 above)
Comment (Justman100) [reply minecraft_servers/archmc/manifest.json]: See 0b335ad

File: minecraft_servers/archmc/manifest.json (hunk 1 above)
Comment (Justman100) [reply minecraft_servers/archmc/manifest.json]: See 0b335ad

File: minecraft_servers/archmc/logo.png
Comment (Justman100) [reply minecraft_servers/archmc/logo.png:1]: See 0b335ad

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**